crates/metaslang/graph_builder/src/variables.rs

```diff
@@ -67,7 +67,7 @@ impl<'a, V> VariableMap<'a, V> {
         }
     }
 
-    /// Clears this enviroment.
+    /// Clears this environment.
```

Comment:
This file is from a forked crate. I suggest minimizing changes to this crate, to prevent future conflicts.